package time_msgs.msgs;


message CheckTime
{

    required bool aux = 1;
  
}
